.github/workflows/build.yaml

```diff
@@ -0,0 +1,45 @@
+name: build
+
+on: push
+
+jobs:
+  build:
+    runs-on: ubuntu-20.04
+
+    env:
+      CGO_ENABLED: "1"
+
+    steps:
+      - name: Setup Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: ^1.15
+        id: go
+
+      - name: Checkout Code
+        uses: actions/checkout@v2
+
+      - name: Install Libraries
```

Comment:
We can simplify this job and eliminate this Install Libraries step if we just build using the docker containers. so, instead of using `make build` we would use `make build-docker`

Reply:
Running tests requires BCC (has to be built from source), so I believe, we have to install the dependencies.

Reply:
My reservation with this approach is 
1. building bcc from source every time is inconsistent, when we release we use a specific commit of bcc that we know works for us.
2. It takes a while to install all these packages and if we use a docker image its all contained in the image
3. Potential code duplication between this script and the Dockerfile.

This is why I made the suggestion. You are correct that to test we will need bcc, but we can also test in a container. For now the tests are broken anyway, and also there's WIP to remove bcc from Tracee, so we can defer the decision for later. 
For now I think it would be best to do a build-docker in this WF and later we can improve it if needed (I also want to merge this by the end of the month to make you eligible for hacktoberfest)

Reply:
Got your point! Will make the required changes.